wordpress-mobile/WordPress-Android
# Issue/6506 google login email

### Fix
Add Google login label, icon, and button to the existing email screen as described in https://github.com/wordpress-mobile/WordPress-Android/issues/6506.  See the screenshots below for details.

![auth_screenshot_02_email](https://user-images.githubusercontent.com/3827611/29277918-6eeaaf12-80d0-11e7-8b21-d436502dab97.png)

### Test
0. Clear app data.
1. Tap ***Log In*** button.
2. Notice Google login view.
3. Rotate device into landscape orientation.
4. Notice Google login view.
